unumpy/_multimethods.py

```diff
@@ -827,29 +827,29 @@ def broadcast_to(array, shape, subok=False):
 
 def _normalize_axis(ndim, axis, allow_repeats=False):
     if isinstance(axis, numbers.Number):
-        axis = (axis,)
-
-    axis_pos = [
-        ndim + operator.index(ax) if ax < 0 else operator.index(ax) for ax in axis
-    ]
+        axis_pos = ndim + operator.index(axis) if axis < 0 else operator.index(axis)
 
-    if not allow_repeats and (len(axis) != len(set(axis_pos))):
-        raise ValueError("Repeated axis.")
-
-    for i, ax in enumerate(axis_pos):
-        if ax >= ndim or ax < 0:
+        if axis_pos >= ndim or axis_pos < 0:
             raise ValueError(
-                "Axis %s is out of bounds for array of dimension %s." % (axis[i], ndim)
+                "Axis %s is out of bounds for array of dimension %s." % (axis, ndim)
             )
 
-    return tuple(axis_pos)
+        return axis_pos
+
+    if isinstance(axis, (tuple, list)):
```

Comment:
Given the above comment, this statement can be removed and everything below de-indented.

Reply:
Also need to check nesting after... To make sure there aren't multiple levels of sequences.

Reply:
> Given the above comment, this statement can be removed and everything below de-indented.

Yes, It makes it a lot cleaner!